Automattic/jetpack
# Add/growth icon to upsell

## Proposed changes:

* Add growth icon to recommendation upsell

### Other information:


## Jetpack product discussion

P2: pbNhbs-bxX-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Connect your user account
3. Go to `/wp-admin/admin.php?page=jetpack#/recommendations/newsletter`
4. Make sure you see the Growth icon
![image](https://github.com/user-attachments/assets/f1eb509d-361a-4146-ac55-e52aa02ef3b4)
